docs/docs/rest-api/public/api/v2/types/pod.raml

```diff
@@ -0,0 +1,80 @@
+#%RAML 1.0 DataType
+type: object
+description:|
+  A pod allows one to launch a collection co-located (on the same agent) containers
+  that share the same network namespace and that may share the same volumes.
+  Resources are specified on a per-container basis.
+properties:
+  id:
+    type: !include pathId.raml
+    description:|
+      Unique identifier for the pod consisting of a series of names separated by slashes.
+      Each name must be at least 1 character and may only contain digits (`0-9`), dashes
+      (`-`), dots (`.`), and lowercase letters (`a-z`). The name may not begin or end with a dash.
+    required: true
+    example: /ops/audit
+  containers:
+    type: array
+    items: !include mesosContainer.raml
+  instances:
+    type: integer
+    description: The number of instances of this pod to start.
+    minimum: 0
+  maxInstances?:
```

Comment:
no it's meant to replace the SINGLE_APP_INSTANCE label. i'm adding upgradestrategy in a follow up commit